### ISSUE TITLE
why did u add this kind of dropout in every residual block

### ISSUE THREAD

**Issue by xiongma** [minsangkim142/Fast-Reading-Comprehension#56]
`def layer_dropout(inputs, residual, dropout):
    pred = tf.random_uniform([]) < dropout
    return tf.cond(pred, lambda: residual, lambda: tf.nn.dropout(inputs, 1.0 - dropout) + residual)`

**Comment by xiongma** [comment]
@localminimum hope you can reply this, thx
